docs/rules/newline-before-return.md

```diff
@@ -1,4 +1,4 @@
-# Require newline before `return` statement (newline-before-return)
+# require an empty line before `return` statements (newline-before-return)
 
 There is no hardfast rule about whether newlines should precede `return` statements in JavaScript. However, clearly delineating where a function is returning can greatly increase the readability and clarity of the code. For example:
```

Comment:
Suggest “empty lines” instead of “newlines” because a newline does precede `return` in the **incorrect** code.